pkg/dependency/parser/gradle/lockfile/parse.go

```diff
@@ -35,10 +35,23 @@ func (Parser) Parse(r xio.ReadSeekerAt) ([]ftypes.Package, []ftypes.Dependency,
 
 		name := strings.Join(dep[:2], ":")
 		version := strings.Split(dep[2], "=")[0] // remove classPaths
+
```

Comment:
Can you please see if `strings.Cut` works in this case?

```suggestion
		version, classPathsString, _ := strings.Cut(dep[2], "=")
```

Reply:
Right, refactored in latest commit, thanks for this suggestion